### PR TITLE
[ZEPPELIN-4299] disable autocomplete in note-name-filter and interpreter search

### PR DESCRIPTION
**Pull request by woowahan-jaehoon**
### What is this PR for?
In chrome browser,
After login with saved-autocomplete feature in chrome and input username in note filter then refresh, note filter is filled username automatically.
It also happen in interpreter page.

this happening with just chrome.
in past, it is not happening this in chrome.

I think that disable auto-complete feature in note filter is better.

### What type of PR is it?
[Improvement]

### Todos
* [x] - modify

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-4299

### How should this be tested?
* Login with autocomplete feature in chrome
* input username in note filter
* refresh
* refresh In interpreter page

### Questions:
* Does the licenses files need update?
    * No
* Is there breaking changes for older versions?
    * No
* Does this needs documentation?
    * No

### CODE REVIEW

**Comment by liuxunorg** [comment]
I am also often troubled by this problem. Thank you for solving this problem.

File: zeppelin-web/src/components/note-name-filter/note-name-filter.html [hunk 1]
```diff
@@ -16,4 +16,4 @@
        ng-click="$event.stopPropagation()"
        placeholder="&#xf002 Filter"
        ng-model="$parent.query.q"
-       ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 300, 'blur': 0 } }" />
+       ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 300, 'blur': 0 } }" autocomplete="off" />
```
Comment (liuxunorg) [inline zeppelin-web/src/components/note-name-filter/note-name-filter.html]: Is it `autocomplete` or `ng-autocomplete`?

**Comment by zjffdu** [comment]
Thanks @woowahan-jaehoon for pointing out this issue. I believe this is regression issue caused by other PR. I am fine to disable it considering the frontend is too old.

**Comment by weand** [comment]
same issue on interpreter page. would you mind addressing this as well?

**Comment by zjffdu** [comment]
ping @woowahan-jaehoon

**Comment by woowahan-jaehoon** [comment]
I tested before create this ticket, It looked like work but I found It's not enough...
(Zeppelin 0.7 also same issue... I think that `ng-autocomplete` has same but I'll test)

It needs more test.

**Comment by zjffdu** [comment]
Thanks @woowahan-jaehoon Looking forward your further update.

**Comment by woowahan-jaehoon** [comment]
I found solution.
In `form` tag, `input` tag is separated by login's input tag.

Adding `autocomplete=off` attribute in `form` tag is more forcing method, but it's enough to solve.

**Comment by zjffdu** [comment]
Awesome, great progress @woowahan-jaehoon

**Comment by zjffdu** [comment]
ping @woowahan-jaehoon Any update on this PR ?

**Comment by woowahan-jaehoon** [comment]
Nop! I think that it is enough.

**Comment by zjffdu** [comment]
Thanks for confirmation, will merge soon

**Comment by weand** [comment]
again faced 'autocomplete with username'-issue on `dependency`-> `artifact` Field on interpreter settings form.

**Comment by woowahan-jaehoon** [comment]
> again faced 'autocomplete with username'-issue on `dependency`-> `artifact` Field on interpreter settings form.

Oh, I didn't check that field. I missed.

I succeeded in reproducing.

**Pull request closed without merging**